packages/components/index.d.ts

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { ReactNode } from 'react';
+
+export interface ViewportTrackerProps {
+    children: React.ReactNode
```

Comment:
`tabWidth` is set to 2 for JS files. It would be great if we can stick to the same for this file as well.